windows/Picker/RNCPickerComponentView.cpp

```diff
@@ -0,0 +1,220 @@
+    packageBuilder,
+    [](const winrt::Microsoft::ReactNative::Composition::IReactCompositionViewComponentBuilder &builder) {
+      builder.as<winrt::Microsoft::ReactNative::IReactViewComponentBuilder>().XamlSupport(true);
+      // Use SetContentIslandComponentViewInitializer
+      builder.SetContentIslandComponentViewInitializer(
+        [](const winrt::Microsoft::ReactNative::Composition::ContentIslandComponentView &islandView) noexcept {
+          const auto userData = winrt::make_self<winrt::Picker::implementation::RNCPickerComponentView>();
+          userData->InitializeContentIsland(islandView);
+          islandView.UserData(*userData);
+        });
+      // Set up initial state with zero size
+      builder.as<winrt::Microsoft::ReactNative::IReactViewComponentBuilder>().SetInitialStateDataFactory(
+        [](const winrt::Microsoft::ReactNative::IComponentProps& /*props*/) noexcept {
+          return winrt::make<RNCPickerStateData>(winrt::Windows::Foundation::Size{ 0, 0 });
+        });
+
+      // Register the measure function - reads from state
+      builder.as<winrt::Microsoft::ReactNative::IReactViewComponentBuilder>().SetMeasureContentHandler(
+        [](winrt::Microsoft::ReactNative::ShadowNode const& shadowNode,
+           winrt::Microsoft::ReactNative::LayoutContext const& ,
+           winrt::Microsoft::ReactNative::LayoutConstraints const&) noexcept {
+
+          const auto currentState = winrt::get_self<RNCPickerStateData>(shadowNode.StateData());
+
+          if (currentState && currentState->desiredSize.Width > 0) {
+            // Return the measured size from state
+            return currentState->desiredSize;
+          }
+
+          // Return a default size if we don't have a measurement yet
+          return winrt::Windows::Foundation::Size{ 100, 32 };
+        });
+
+      // Handle state updates
+      builder.as<winrt::Microsoft::ReactNative::IReactViewComponentBuilder>().SetUpdateStateHandler(
+        [](const winrt::Microsoft::ReactNative::ComponentView& view,
+           const winrt::Microsoft::ReactNative::IComponentState& newState) {
+          const auto islandView = view.as<winrt::Microsoft::ReactNative::Composition::ContentIslandComponentView>();
+          const auto userData = islandView.UserData().as<winrt::Picker::implementation::RNCPickerComponentView>();
+          userData->UpdateState(view, newState);
+        });
+    });
+}
+
+void RNCPickerComponentView::InitializeContentIsland(
+  const winrt::Microsoft::ReactNative::Composition::ContentIslandComponentView &islandView) {
+  // Create ComboBox for picker functionality
+  m_comboBox = winrt::Microsoft::UI::Xaml::Controls::ComboBox();
+  m_comboBox.HorizontalAlignment(winrt::Microsoft::UI::Xaml::HorizontalAlignment::Stretch);
+  
+  // For editable ComboBox, trigger selection change always (not just on commit)
+  m_comboBox.SelectionChangedTrigger(winrt::Microsoft::UI::Xaml::Controls::ComboBoxSelectionChangedTrigger::Always);
```

Comment:
```suggestion
  m_comboBox.HorizontalAlignment(
    winrt::Microsoft::UI::Xaml::HorizontalAlignment::Stretch);
  
  // For editable ComboBox, trigger selection change always (not just on commit)
  m_comboBox.SelectionChangedTrigger(
    winrt::Microsoft::UI::Xaml::Controls::ComboBoxSelectionChangedTrigger::Always);
```

Minor: fix formatting